DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: scoringutils
 Title: Utilities for Scoring and Assessing Predictions
-Version: 1.1.4
+Version: 1.1.5
```

Comment:
Do you want to identify that this is a development release or is this a release as is?

Reply:
So the end number here is the minor release number. EpiNow2 has a dev number. If you use that approach then for example pkgdown will build a dev version or a released version of the package. It is also clear when a release (in this case 1.1.5 is done or if its still underwork). At the moment here people could install the same release but get different behaviour. I don't feel that strongly here so really just an FYI.

Reply:
@nikosbosse resolve as you wish

Reply:
Just had a look at this: https://combine-australia.github.io/r-pkg-dev/versioning.html
There is a second PR coming once that one is merged to add some documentation, I suggest keeping this for now and making it a minor release in the second PR.